go.mod

```diff
@@ -11,5 +11,6 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.10.1
 	github.com/tendermint/tendermint v0.34.15 // indirect
+	github.com/twilio/twilio-go v0.26.0 // indirect
```

Comment:
did you run `go mod tidy`? twilio-go should not be an indirect module dependency.